### PR TITLE
Show off FallbackResource in Alias Paths To Script section

### PR DESCRIPTION
**Pull request by djc**

### CODE REVIEW

**Comment by phanan** [comment]
I'm not sure I follow what you mean by 

> This is a less efficient version of the FallbackResource directive (because using `mod_rewrite` is more complex than just handling the `FallbackResource` directive), but it's also more flexible.

Care to explain a bit further?

**Comment by djc** [comment]
Which part do you find hard to follow? It seems clear to me that `FallbackResource` can just forward the request straight to the right handler, whereas `mod_rewrite` has to do a bunch of processing on the request before it can decide how to handle the request.

**Comment by phanan** [comment]
Sorry, I meant "less efficient" and "but also more flexible" can't really go together can they? I'm no native speaker though.

**Comment by djc** [comment]
Sure they can. When using `mod_rewrite`, Apache uses more resources to process the request, but `mod_rewrite` also allows you to redirect requests in many different ways by doing different kinds of processing on it. Efficient therefore pertains to the amount of processing time spent on requests (more when using `mod_rewrite` -- so it's less efficient), flexible pertains to the different ways you can process request (also more when using `mod_rewrite` -- but this is a good thing). Hope that helps!

**Comment by phanan** [comment]
It does. Thanks!
